examples/hubert/dataset/hubert_dataset.py

```diff
@@ -276,13 +276,14 @@ def __call__(self, batch: Tuple[Tensor, Tensor, int]) -> Tuple[Tensor, Tensor, T
             waveforms.append(waveform)
             lengths.append(length)
             labels.append(label)
-
-        data = torch.zeros(len(batch), audio_size)
-        for i in range(len(waveforms)):
-            data[i][0 : waveforms[i].shape[1]] = waveforms[i][0]
+        if self.pad:
+            waveforms = torch.nn.utils.rnn.pad_sequence(waveforms, batch_first=True)
+            labels = torch.nn.utils.rnn.pad_sequence(labels, batch_first=True)
+        else:
+            waveforms = torch.stack(waveforms)
+            labels = torch.stack(labels)
         lengths = torch.tensor(lengths)
-        labels = torch.nn.utils.rnn.pad_sequence(labels, batch_first=True)
-        return data, labels, lengths
+        return waveforms, labels, lengths
 
     def _collate_audio_label(
```

Comment:
"collate" doesn't accurately describe what the function does. perhaps `_crop_audio_label`?